batch/batch/cloud/azure/driver/create_instance.py

```diff
@@ -250,12 +253,13 @@ def create_vm_config(
 -v /batch:/batch:shared \
 -v /logs:/logs \
 -v /global-config:/global-config \
--v /gcsfuse:/gcsfuse:shared \
+-v /cloudfuse:/cloudfuse:shared \
```

Comment:
I don't understand this bit. Why do we mount `/cloudfuse` from the VM into the *worker* container at `/cloudfuse`? Neither the VM nor the worker container use cloud fuse, right?

Reply:
The batch worker creates directories for a specific job in /cloudfuse. This needs to be on the host's file system.